pkg/platform/snaprepo/s3_config.go

```diff
@@ -125,51 +125,51 @@ type S3TypeConfig struct {
 
 // Validate ensures that S3Config is
 func (c S3Config) Validate() error {
-	var merr = new(multierror.Error)
+	var merr = multierror.NewPrefixed("s3 configuration")
 	if err := validateRequiredSettings(c); err != nil {
-		merr = multierror.Append(merr, err)
+		merr = merr.Append(err)
 	}
 	if e := validateOptionalSettings(c); e != nil {
-		merr = multierror.Append(merr, e)
+		merr = merr.Append(e)
 	}
 
 	return merr.ErrorOrNil()
 }
 
 func validateRequiredSettings(c S3Config) error {
-	var err = new(multierror.Error)
+	var merr = multierror.NewPrefixed("required setting")
```

Comment:
For consistency. No other commands have a "required setting" prefix for when validation is done
```suggestion
	var merr = multierror.NewPrefixed("s3 configuration")
```